src/styles/app/17-main-tabs.css

```
@@ -444,20 +445,33 @@
 }
 
 .mainTab:hover {
 	color: var(--text-primary);
 }
 
 .mainTab:active {
-	opacity: 0.85;
+	transform: scale(0.96);
+}
+
+@keyframes mainTabZoomIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "mainTabZoomIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)